data/future.ts

```diff
@@ -1,73 +1,99 @@
 let POST_GRAD = [{
-  "name": "Have offer",
-  "value": 43
+  "name": "Entrepreneurship",
+  "value": 2
   }, {
   "name": "Grad school",
-  "value": 3
+  "value": 1
   }, {
-  "name": "Accepted offer",
-  "value": 44
+  "name": "Taking a break",
+  "value": 1
   }, {
-  "name": "Have offer, not interviewing",
-  "value": 5
+  "name": "Industry - Have Offer(s)",
+  "value": 1
   }, {
-  "name": "No offer, interviewing",
-  "value": 17
+  "name": "Industry - Accepted",
+  "value": 49
 }];
 
+
 let POST_LOCATION = [{
-  "name": "California",
-  "value": 49
+  "name": "US - California",
+  "value": 17
   }, {
-  "name": "West Coast Canada",
-  "value": 4
+  "name": "Canada - West Coast",
+  "value": 1
   }, {
-  "name": "East Coast Canada",
-  "value": 30
+  "name": "Canada - East Coast",
+  "value": 11
   }, {
-  "name": "East Coast US",
-  "value": 16
+  "name": "US - East Coast",
+  "value": 18
   }, {
-  "name": "US Pacific North West",
-  "value": 11
+  "name": "US - Pacific North West",
+  "value": 4
   }, {
-  "name": "Asia",
-  "value": 2
+  "name": "US - Midwest",
+  "value": 1
 }]
 
 let MOTIVATIONS = [{
-  "name": "Compensation",
-  "value": 53
+  "name": "Good Culture/Team/Coworkers",
+  "value": 40
   }, {
-  "name": "Family",
-  "value": 19
+  "name": "Compensation",
+  "value": 35
   }, {
-  "name": "Good coworkers",
-  "value": 34
+  "name": "Location",
+  "value": 31
   }, {
-  "name": "Having an impact",
-  "value": 26
+  "name": "Having an Impact",
+  "value": 22
   }, {
-  "name": "Location",
-  "value": 30
+  "name": "Improving Skills",
+  "value": 23
   }, {
-  "name": "Improving skills",
-  "value": 39
+  "name": "Job security due to pandemic",
```

Comment:
since the other labels are in title caps, should this also be title caps as well?
```suggestion
  "name": "Job Security Due to Pandemic",
```